packages/__docs__/src/Returns/index.tsx

```diff
@@ -29,7 +29,8 @@ import { Table } from '@instructure/ui-table'
 import { compileMarkdown } from '../compileMarkdown'
 
 import type { ReturnsProps } from './props'
-import { JSDocFunctionReturns } from '../../buildScripts/DataTypes'
+// eslint-disable-next-line import/no-unresolved
+import type { JSDocFunctionReturns } from '../../buildScripts/DataTypes'
```

Comment:
The linter has found a couple of errors, this is an ugly workaround. Its getting confused because the docs build scripts are compiled to ES6 modules and the docs app to commonJS. IMO we should not bother much with this, rather move the docs app to a modern build system like ESBuild